app/helpers/application_helper.rb

```diff
@@ -1,4 +1,9 @@
 module ApplicationHelper
+  ### TEMP TEMP TEMP
+  def videojs_rails(*_args)
+    content_tag(:p, 'Sorry, this video is unavailable', class: 'word_gloss', style: 'min-height: 150px')
```

Comment:
can you call `t()` here - most of the app already is translatable and maybe we can do Te Reo one day.